vending-app/src/main/java/com/android/vending/Util.kt

```diff
@@ -0,0 +1,45 @@
+package com.android.vending
+
+import android.accounts.Account
+import android.accounts.AccountManager
+import android.accounts.AccountManagerFuture
+import android.os.Bundle
+import android.util.Log
+import java.io.ByteArrayOutputStream
+import java.io.IOException
+import java.util.zip.GZIPOutputStream
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+import kotlin.coroutines.suspendCoroutine
+
+private const val TAG = "FakeStoreUtil"
+
+/**
+ * From [StackOverflow](https://stackoverflow.com/a/46688434/), CC BY-SA 4.0 by Sergey Frolov, adapted.
+ */
+fun ByteArray.encodeGzip(): ByteArray {
```

Comment:
Should remain in LicenseRequestHeaders.kt  as it is not used anyway else for now ? 

Util classes usually end up in tote bag classes and become a mess.